From cd525502af07910a36166eac051fc94cd46e2e8f Mon Sep 17 00:00:00 2001
From: Robin Gong <b38343@freescale.com>
Date: Thu, 13 Aug 2015 16:39:06 +0800
Subject: [PATCH 0397/1691] MLK-11344-11: spi: spi-imx: enable dma support for
 i.mx6dl

commit cd525502af07910a36166eac051fc94cd46e2e8f from
git://git.freescale.com/imx/linux-2.6-imx.git imx_4.1.15_1.0.0_ga

There is official workaound for TKT238285, so remove the limitation
for i.mx6dl.

Signed-off-by: Robin Gong <b38343@freescale.com>
---
 drivers/spi/spi-imx.c | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/drivers/spi/spi-imx.c b/drivers/spi/spi-imx.c
index 8f3e273..0c491c8 100644
--- a/drivers/spi/spi-imx.c
+++ b/drivers/spi/spi-imx.c
@@ -843,10 +843,6 @@ static int spi_imx_sdma_init(struct device *dev, struct spi_imx_data *spi_imx,
 {
 	int ret;
 
-	/* use pio mode for i.mx6dl chip TKT238285 */
-	if (of_machine_is_compatible("fsl,imx6dl"))
-		return 0;
-
 	/* Prepare for TX DMA: */
 	master->dma_tx = dma_request_slave_channel(dev, "tx");
 	if (!master->dma_tx) {
-- 
1.9.1

